utils.js

```diff
@@ -643,6 +648,68 @@ function getAllocationsFromTx (tx) {
   return syscointx.getAllocationsFromTx(tx)
 }
 
+/* isBech32
+Purpose: Return a boolean if a given sys address is a bech32 address
+Param address: Required. Address to check
+*/
+function isBech32 (address) {
+  try {
+    utxoLib.address.fromBech32(address)
+    return true
+  } catch (e) {
+    return false
+  }
+}
+
+/* isScriptHash
+Purpose: Return  a boolean if a given sys address is a script hash accordingly to the syscoinNetwork selected
+Param address: Required. Address to check
+Param coinInfo: Required. networkInfo
```

Comment:
networkInfo rename

Reply:
I don't see this fixed?

Reply:
it is updated already, check on the new version: 
https://github.com/claudiovb/syscoinjs-lib/blob/0cb9dd6c3518cbb1411a4b0f5e8974d57b3418e5/utils.js#L667